test/release/examples/benchmarks/shootout/fasta.chpl

```diff
@@ -133,7 +133,7 @@ proc randomMake(desc, a, n) {
         line_buff[i] = a[hi](nucl);
       }
     }
-    line_buff[bytes] = newline;
+    line_buff[bytes] = newline:int(8);
```

Comment:
I tried that but wasn't able to get it to propagate through the compiler, I believe for reasons related to why this cast is needed:  I don't think we support good coercions from enums to uint(8)s (yet... I think this has been on my TODO list for years now).  Are you able to create a proposed version of the program that works and is clean?  Could we take the idealized version of fasta into a separate PR?